TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md

```diff
@@ -0,0 +1,175 @@
+
+
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+
+
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+
+
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+## Agenda and Minutes
+
+
+### Welcome
+
+
+* Dan Middleton (DM) opened the call at 7:04am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Helen Lau (HL) assisted with the meeting minutes
+* DM reviewed the agenda
+
+
+### Roll Call / Attendance
+
+
+***Voting members of the TAC***
+
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+
+* Accenture - Giuseppe Giordano 
+* Arm - Nathaniel McCallum  / Michael Lu
+* Google - Cfir Cohen  / Catalin Sandu +
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Dave Thaler + / Alec Fernandez + 
+* Red Hat - Lily Sturmann + / Yash Mankad +
+
+
+   " + " indicates attending
+
+
+***Other Attendees***
+
+
+* Chris Ramming (VMware)
+* Elaine Shi (Carnegie Mellon University)
+* Eric Voit (Cisco)
+* Kanth Ghatraju (Oracle) 
+* Mark Novak (JP Morgan Chase)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Victor Lu
+* Ye Li (VMware)
+
+
+* Ben Sternthal (LF)
+* Helen Lau (LF)
+
+
+**Welcome New Community Members**
+
+
+* Chris Ramming - VMWare: Common developer API for confidential computing. VMWare will introduce a project to the TAC mailing list in two weeks.
+
+
+
+
+**Meeting Minute Approval**
+
+
+Proposed: That the minutes of the September 7, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+
+
+
+** Congratulations to rust-spdm**
+
+
+* TAC Voting members voted to approve rust-spdm as a new project.
+
+
+
+
+**Tech Talk: Oblivious RAM - Prof. Elaine Shi**
+
+
+* Prof. Elaine Shi presented Oblivious RAM.
+* Oblivious RAM PowerPoint: https://drive.google.com/file/d/1S8JWXOTUMQQKocE4AOMrMWndYK31nRI4/view?usp=sharing
+* Currently working on counterpart of the STL library.
+* Prof. Shi plans to form a non-profit to continue work on Oblivious RAM and would like to connect with the CCC.
+
+
+
+
+**2023 TAC Goals: Cross project collab survey.**
+
+
+* Eric Voit (EV) gave an update on the cross project collab survey that was distributed to project mentors. 
+* The goal is to drive project mentor interest on shared goals and code dependencies within their projects. There have been minimal responses to the survey from project mentors. 
+
+
+*** Action item to all Project Mentors: Take the cross project collab survey. ***

```

Comment:
I thought the action item was for the mentors to get the projects to respond to the survey (as opposed to the mentors responding to the survey)

Reply:
Hi Dave. I followed up with Lily following the TAC last week, and she sent it to the project mentors. Unless Lily advises that she'd like to send it to the projects for a response, I'll leave this action item as is for the minutes.

Reply:
That's not my point.

```suggestion
*** Action item to all Project Mentors: Get the projects to take the cross project collab survey. ***

```